packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -176,7 +176,13 @@ export class SpinButton extends BaseComponent<ISpinButtonProps, ISpinButtonState
                 autoComplete="off"
                 role="spinbutton"
                 aria-labelledby={label && this._labelId}
-                aria-valuenow={ariaValueNow ? ariaValueNow : !isNaN(Number(value)) ? Number(value) : undefined}
+                aria-valuenow={
+                  ariaValueNow !== undefined && !isNaN(ariaValueNow)
```

Comment:
could you do !isNaN(Number(ariaValueNow)) instead of having to check for undefined?